src/scss/06_components/paragraphs/_service-list.scss

```diff
@@ -8,44 +8,54 @@
   }
 
   .views-exposed-form {
+    border-bottom: 1px solid $color-black-10;
```

Comment:
It seems that this selector was a left over copy-paste mistake by us, so you're free to use `.views--service-list .views-exposed-form` and remove any unnecessary code bits from it.